docs/classic-ui/static-resources.md

```diff
@@ -100,4 +100,20 @@ The following attributes are available for registering a static resource:
 
 ## Loading order of resources
 
-`depends` is used to define the loading order of resources by specifying the name of the depending bundle.
+You can use the `depends` attribute to define the order in which resources load.
+
+You can specify a list of either the names of the bundles on which this bundle depends or a single list item of `["all"]`.
+In the latter case of `["all"]` for a bundle, all other JavaScript and CSS resources in their resource groups will render before this bundle, making it the final bundle to render in its resource group.
```

Comment:
What happens if there are two bundles depending on "all"? Is this prevented? If not, is there some order or is it random?

Reply:
I'd say then the alphabetical sorting in the resource registry takes effect, which you could manipulate with special prefixes (eg. "zzz_mybundle")

Reply:
If more bundles depend on "all", all the "all" bundles are rendered in the order they were defined

Reply:
Addressed in f80c6103029f37c3e9bfb9c5fc70292f36442881